### PR TITLE
(WIP) Revert apc_chat translations to pre-forced translations

### PR DESCRIPTION
**Pull request by VariantXYZ**

### CODE REVIEW

**Comment by VariantXYZ** [comment]
@alama want to fix all my spelling errors

File: Dialogue/apc_chat.csv [hunk 1]
```diff
@@ -3634,73 +3634,73 @@ IncidentJsApcBegin#47,"""/ci2 2 よっ、大丈夫か！<br>こんな格好で
...
+DamageLarge#56,"""Urgh!"""
+Trap#55,"""Cowardly traps!"""
+ThankTrapRescue#55,"""Thanks!"""
+TrapRescueOther#54,"""We don't have time to waste!"""
+MemberDamaged#54,"""Leave it to me!"""
+MemberDead#55,"""Unforgivable!"""
+AbilityExe#54,"""/toge I'm in top form!"""
+AbilitySpecial#54,"""/toge Phew!\u3000Leave it to me!"""
+Resurrect#54,"""Back in shape!"""
+PartyJoin#54,"""/ci1 1 Happy to come along!"""
```
Comment (SynthSy) [inline Dialogue/apc_chat.csv]: It becomes wrong from here on out.

**Comment by VariantXYZ** [comment]
@SynthSy to summarize: this file is the 'cleanest' we've got, but it's still got a few errors that have persisted since the initial commit. 

I'll leave it to you to decide what needs fixing and if you'd prefer to instead start from scratch.

**Comment by alama** [comment]
Can not merge EN-v50200_rc_97

**Pull request closed without merging**